bin/_mocha

```diff
@@ -68,8 +68,8 @@ program
   .option('-S, --sort', "sort test files")
   .option('-b, --bail', "bail after first test failure")
   .option('-d, --debug', "enable node's debugger, synonym for node --debug")
-  .option('-g, --grep <pattern>', 'only run tests matching <pattern>')
-  .option('-f, --fgrep <string>', 'only run tests containing <string>')
+  .option('-g, --grep <pattern>', 'only run tests having full title matching <pattern>')
+  .option('-f, --fgrep <string>', 'only run tests having full title containing <string>')
```

Comment:
I don't think this is correct. If you want exact matching you have to use `^full title$'. Also this PR doesn't change the way the matching works.